### PR TITLE
Api v4.0.4

### PR DESCRIPTION
**Pull request by menduz**

### CODE REVIEW

File: _posts/sdk-reference/2018-01-21-scene-content-guide.md [hunk 1]
```diff
@@ -36,16 +36,29 @@ Any of these can be added to your scene, they can all include basic components l
 
 See [Entity interfaces]({{ site.baseurl }}{% post_url /sdk-reference/2018-06-21-entity-interfaces %}) for more details on these types of entities.
 
+<<<<<<< HEAD
```
Comment (menduz) [inline _posts/sdk-reference/2018-01-21-scene-content-guide.md]: Merge conflict

File: _posts/sdk-reference/2018-01-21-scene-content-guide.md [hunk 2]
```diff
@@ -36,16 +36,29 @@ Any of these can be added to your scene, they can all include basic components l
...
 > Tip: When editing the code via a source code editor (like Visual Studio Code), you can see the list of components supported by a type of entity. Typically, this is done by placing the cursor in the entity and typing _Ctrl + Space bar_.
+>>>>>>> master
 
 ## Entity positioning
 
-All entities can have a position, a rotation and a scale. These can be easily set as components, as shown below:
+All entities can have a *position*, a *rotation* and a *scale*. These can be easily set as components, as shown below:
 
 {% raw %}
+<<<<<<< HEAD
```
Comment (menduz) [inline _posts/sdk-reference/2018-01-21-scene-content-guide.md]: merge conflict

File: _posts/sdk-reference/2018-01-21-scene-content-guide.md [hunk 3]
```diff
@@ -59,52 +72,89 @@ When an entity is nested inside another, the child entities inherit components f
 You can include an invisible base entity to wrap a set of other entities and define their positioning as a group.
 
 {% raw %}
+<<<<<<< HEAD
```
Comment (menduz) [inline _posts/sdk-reference/2018-01-21-scene-content-guide.md]: merge conflict

File: _posts/sdk-reference/2018-01-21-scene-content-guide.md [hunk 4]
```diff
@@ -59,52 +72,89 @@ When an entity is nested inside another, the child entities inherit components f
...
+  </entity>
+=======
 
 ```tsx
 <entity position={{ x: 0, y: 0, z: 1 }} rotation={{ x: 45, y: 0, z: 0 }}>
   <box position={{ x: 10, y: 0, z: 0 }} scale={2} />
   <box position={{ x: 10, y: 10, z: 0 }} scale={1} />
   <box position={{ x: 0, y: 10, z: 0 }} scale={2} />
 </entity>
+>>>>>>> master
```
Comment (menduz) [inline _posts/sdk-reference/2018-01-21-scene-content-guide.md]: merge conflict

File: _posts/sdk-reference/2018-01-21-scene-content-guide.md [hunk 5]
```diff
@@ -59,52 +72,89 @@ When an entity is nested inside another, the child entities inherit components f
...
+```tsx
+ <box 
+    rotation={currentRotation}
+    transition={{ rorotation: { duration: 1000, timing: "ease-in" }}}
+  />
+=======
 
 ```tsx
 <box rotation={currentRotation} transition={{ rorotation: { duration: 1000, timing: "ease-in" } }} />
+>>>>>>> master
```
Comment (menduz) [inline _posts/sdk-reference/2018-01-21-scene-content-guide.md]: merge conflict

File: _posts/sdk-reference/2018-01-21-scene-content-guide.md [hunk 6]
```diff
@@ -59,52 +72,89 @@ When an entity is nested inside another, the child entities inherit components f
...
+* rotation
+* scale
+* color
+* lookAt
+=======
 - position
 - rotation
 - scale
 - color
+>>>>>>> master
```
Comment (menduz) [inline _posts/sdk-reference/2018-01-21-scene-content-guide.md]: merge conflict

File: _posts/sdk-reference/2018-01-21-scene-content-guide.md [hunk 7]
```diff
@@ -59,52 +72,89 @@ When an entity is nested inside another, the child entities inherit components f
...
+    transition={ 
+=======
 
 ```tsx
  <box
     rotation={currentRotation}
     color={currentColor}
     scale={currentScale}
     transition={
+>>>>>>> master
```
Comment (menduz) [inline _posts/sdk-reference/2018-01-21-scene-content-guide.md]: merge conflict

File: _posts/sdk-reference/2018-01-21-scene-content-guide.md [hunk 8]
```diff
@@ -123,6 +173,10 @@ The transition component allows you to set:
 In the example below, a transition is applied to the rotation of an invisible entity that wraps a box. As the box is off-center from the parent entity, the box pivots like an opening door.
 
 {% raw %}
+<<<<<<< HEAD
+```tsx
+=======
```
Comment (menduz) [inline _posts/sdk-reference/2018-01-21-scene-content-guide.md]: merge conflict

File: _posts/sdk-reference/2018-01-21-scene-content-guide.md [hunk 9]
```diff
@@ -132,14 +186,66 @@ In the example below, a transition is applied to the rotation of an invisible en
...
+```tsx
+  <sphere 
+    position={{ x: 0.5, y: 1, z: 0 }}   
+    color="#EF2D5E"
+  />
+=======
 
 ```tsx
 <sphere position={{ x: 0.5, y: 1, z: 0 }} color="#EF2D5E" />
+>>>>>>> master
```
Comment (menduz) [inline _posts/sdk-reference/2018-01-21-scene-content-guide.md]: merge conflict

File: _posts/sdk-reference/2018-01-21-scene-content-guide.md [hunk 10]
```diff
@@ -155,21 +261,57 @@ Materials can be applied to primitive entities and to planes, simply by setting
 {% raw %}
 
 ```tsx
+<<<<<<< HEAD
```
Comment (menduz) [inline _posts/sdk-reference/2018-01-21-scene-content-guide.md]: merge conflict

File: _posts/sdk-reference/2018-01-21-scene-content-guide.md [hunk 11]
```diff
@@ -155,21 +261,57 @@ Materials can be applied to primitive entities and to planes, simply by setting
...
+  <basic-material 
+    id="basic_material" 
+    texture="profile_avatar.png"
+  />
+  <sphere
+    material="#basic_material" 
+  />
+```
+=======
+>>>>>>> master
```
Comment (menduz) [inline _posts/sdk-reference/2018-01-21-scene-content-guide.md]: merge conflict

File: _posts/sdk-reference/2018-01-21-scene-content-guide.md [hunk 12]
```diff
@@ -155,21 +261,57 @@ Materials can be applied to primitive entities and to planes, simply by setting
...
 {% endraw %}
 
 Materials are also implicitly imported into a scene when you import a glTF model that includes embedded materials. When that's the case, the scene doesn't need a `<material/>` entity declared.
 
 ## Import 3D models
+<<<<<<< HEAD
+ 
+For more complex shapes, you can build a 3D model in an external tool like Blender and then import them in *glTF* format.  [glTF](https://www.khronos.org/gltf) (GL Transmission Format) is an open project by Khronos providing a common, extensible format for 3D assets that is both efficient and highly interoperable with modern web technologies.
+=======
+>>>>>>> master
```
Comment (menduz) [inline _posts/sdk-reference/2018-01-21-scene-content-guide.md]: merge conflict

File: _posts/sdk-reference/2018-01-21-scene-content-guide.md [hunk 13]
```diff
@@ -180,33 +322,55 @@ To add an external model into a scene, add a `<gltf-model>` element and set its
...
+<gltf-model
+    position={{ x: 5, y: 3, z: 5 }}
+    scale={0.5}
+    src="models/myModel.gltf"
+  />
+=======
 
 ```tsx
 <gltf-model position={{ x: 5, y: 3, z: 5 }} scale={0.5} src="models/myModel.gltf" />
+>>>>>>> master
```
Comment (menduz) [inline _posts/sdk-reference/2018-01-21-scene-content-guide.md]: merge conflict

File: _posts/sdk-reference/2018-01-21-scene-content-guide.md [hunk 14]
```diff
@@ -180,33 +322,55 @@ To add an external model into a scene, add a `<gltf-model>` element and set its
...
 ```
 
 {% endraw %}
 
+<<<<<<< HEAD
+glTF models can have either a `.gltf` or a `.glb` extension. glTF files are human-readable, you can open one in a text editor and read it like a JSON file. This is useful, for example, to verify that animations are properly attached and to check for their names. glb files are binary, so they're not readable but they are considerably smaller in size, which is good for the scene's performance. 
+=======
 glTF models can have either a `.gltf` or a `.glb` extension. glTF files are human-readable, you can open one in a text editor and read it like a JSON file. This is useful, for example, to verify that animations are properly attached and to check for their names. glb files are binary, so they're not readable but they are considerably smaller in size, which is good for the scene's performance.
+>>>>>>> master
```
Comment (menduz) [inline _posts/sdk-reference/2018-01-21-scene-content-guide.md]: merge conflict

File: _posts/sdk-reference/2018-01-21-scene-content-guide.md [hunk 15]
```diff
@@ -180,33 +322,55 @@ To add an external model into a scene, add a `<gltf-model>` element and set its
...
 glTF models can also include their own textures and animations. Keep in mind that all models, their shaders and their textures must be within the parameters of the [scene limitations]({{ site.baseurl }}{% post_url /documentation/building-scenes/2018-01-06-scene-limitations %}).
 
+<<<<<<< HEAD
+
+> Note: Keep in mind that all models and their textures must be within the parameters of the [scene limitations]({{ site.baseurl }}{% post_url /sdk-reference/2018-01-06-scene-limitations %}).
+
+> Note: obj models are also supported as a legacy feature, but will likely not be supported for much longer. To add one, use an `<obj-model>` entity. 
+=======
 > Note: obj models are also supported as a legacy feature, but will likely not be supported for much longer. To add one, use an `<obj-model>` entity.
+>>>>>>> master
```
Comment (menduz) [inline _posts/sdk-reference/2018-01-21-scene-content-guide.md]: merge conflict

File: _posts/sdk-reference/2018-01-21-scene-content-guide.md [hunk 16]
```diff
@@ -180,33 +322,55 @@ To add an external model into a scene, add a `<gltf-model>` element and set its
...
-In a dynamic scene, you reference an animation by its armature name, an underscore and its animation name. For example `myArmature_animation1`. You activate an animation by setting its `playing` property to `true`.
+You activate an animation by adding *skeletalAnimation* settings to a gltf model and setting the `playing` property of one of its clips to `true`.
 
 The example below imports a model that includes animations and configures them:
 
 {% raw %}
+<<<<<<< HEAD
+=======
 
+>>>>>>> master
```
Comment (menduz) [inline _posts/sdk-reference/2018-01-21-scene-content-guide.md]: merge conflict

**Comment by menduz** [comment]
LGTM

File: _posts/documentation/building-scenes/2018-01-03-create-scene.md [hunk 17]
```diff
@@ -127,34 +106,55 @@ reading the `build.json` file and compile your scene in a way that the client ca
...
-## tsconfig.json
+### tsconfig.json
 
 Directories containing a `tsconfig.json` file are root directories for TypeScript Projects. The `tsconfig.json` file specifies the root files and options required to compile your project in JavaScript.
 
 > You can use another tool or language instead of TypeScript, so long as your scripts are contained within a single Javascript file (scene.js). All provided type declarations are made in TypeScript, and other languages and transpilers are not officially supported.
 
 
-## Preview your scene
+## Local scene contents
```
Comment (menduz) [inline _posts/documentation/building-scenes/2018-01-03-create-scene.md]: Local or static?

File: _posts/documentation/building-scenes/2018-01-04-preview-scene.md [hunk 18]
```diff
@@ -0,0 +1,130 @@
...
+
+
+## Preview a remote scene
+
+To preview a remote scene you must first start a WebSockets server in your local machine. To do this:
+
+From the scene directory run the following bash commands:
+
+```bash
+$:  cd server
```
Comment (menduz) [inline _posts/documentation/building-scenes/2018-01-04-preview-scene.md]: Please avoid starting bash commands with `$:` or `$`. That makes no sense in any shell

**Comment by nearnshaw** [comment]
@menduz 
I just made a couple of fixes based on your feedback
ready to merge

**Comment by nearnshaw** [comment]
Weird, we get this error message:

```
Liquid Exception: Could not find post "/documentation/building-scenes/2018-01-08-sample-scenes" in tag 'post_url'. Make sure the post exists and the name is correct. in /tmp/client/_posts/documentation/building-scenes/2018-01-04-preview-scene.md
             ERROR: YOUR SITE COULD NOT BE BUILT:
                    ------------------------------------
                    Could not find post "/documentation/building-scenes/2018-01-08-sample-scenes" in tag 'post_url'. Make sure the post exists and the name is correct. 
```

The reason why this happens is that `/building-scenes/2018-01-08-sample-scenes` no longer exists, the content there is now in its own folder in `/examples` .     There are conflicts surely because of the linting that must have happened on that page this branch.

I'm not sure how to fix this, though

**Comment by menduz** [comment]
fix pushed

**Comment by menduz** [comment]
you need to approve this, I can't since I created the PR

**Review by nearnshaw — APPROVED** [review]

**Pull request merged**